packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -90,6 +90,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
             traceContainerClassName = traceContainerClassName + ' theia-mod-selected';
         }
         return <div className={traceContainerClassName}
+            title={outputName + ': ' + outputDescription}
             id={`${traceContainerClassName}-${props.index}`}
             key={props.key}
             style={props.style}
```

Comment:
Is there supposed to be a tooltip when hovering over a text with the ellipsis? I don't see any.